src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,64 @@
 import React from 'react';
 import { Loader } from '../Loader';
+import { TodoUser } from '../../types/User';
+import { check } from '../../api';
+
+type Props = {
+  user: TodoUser
+  setEyeMark: (arg: number) => void
+  setUserModal: (arg: TodoUser | null) => void
+};
+
+export const TodoModal: React.FC<Props> = ({
+  user,
+  setEyeMark,
+  setUserModal,
+}) => {
+  const closeButtonHandler = () => {
+    setUserModal(null);
+    setEyeMark(-1);
+  };
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
-        <Loader />
-      ) : (
+      { check(user.todo) ? (
         <div className="modal-card">
           <header className="modal-card-head">
             <div
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${user.todo.id}`}
```

Comment:
use destructuring assignment

Reply:
It's impossible because the return consists of the conditional operator. Destruct in arguments is impossible to do because need to assign Props & at the moment when the user object has come to the component null (wait for promise done)